naivedatetimefield/__init__.py

```diff
@@ -135,16 +135,16 @@ def to_python(self, value):
     def get_prep_value(self, value):
         return super(DateTimeField, self).get_prep_value(value)
 
-    def get_db_prep_value(self, value, connection, prepared=False):
-        return super().get_db_prep_value(value, connection, prepared)
```

Comment:
Removing this - it's a straight pass-through, I put it there to aid debugging and forgot to take it out.